aries-site/src/examples/templates/page-layouts/anatomy/PageMarginAnatomy.js

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import { Box, Grid, Stack, Text } from 'grommet';
+import PropTypes from 'prop-types';
+
+// values chosen to best visually represent the margin area in diagram
+const largeColumns = ['13px', 'auto', '13px'];
+const smallColumns = ['8px', 'auto', '8px'];
+
+const MarginText = ({ label }) => (
+  <Stack anchor="bottom">
+    <Text alignSelf="center">{label}</Text>
+    {/* values chosen to best visually represent horizontal line in diagram. */}
+    <Box width="12px" height="1px" background="black" />
+  </Stack>
+);
+
+const GridViewPort = ({ columns, label, marginLabel, width }) => (
+  <Box gap="small" width={width}>
+    <Box direction="row" justify="between">
+      <MarginText label={marginLabel} />
+      <MarginText label={marginLabel} />
+    </Box>
+    <Grid
+      columns={columns}
+      // values chosen to best visually represent the margin area in diagram
+      rows={['20px', 'small', '20px']}
```

Comment:
Can we use theme value edgesize.large here instead? That would equate to 24px -- is it reasonable? or too big?

Reply:
let me check

Reply:
First image is `'24px'`
<img width="791" alt="Screen Shot 2022-01-26 at 4 55 42 PM" src="https://user-images.githubusercontent.com/42451602/151266611-bfa4b365-d464-44fd-abf7-d7ebc9489486.png">

second image with the `20px`
<img width="727" alt="Screen Shot 2022-01-26 at 4 56 21 PM" src="https://user-images.githubusercontent.com/42451602/151266680-66bbe90e-1b58-442b-bece-e62cac4f1f02.png">

Reply:
I think using the theme value is totally fine